### PR TITLE
Include k8s in retrieving active sessions list for web UI

### PR DESCRIPTION
**Pull request by kimlisa**
#### Description
part of https://github.com/gravitational/teleport/issues/14009

The get active sessions web api endpoint now includes listing k8s sessions.

### CODE REVIEW

File: lib/web/apiserver.go [hunk 1]
```diff
@@ -2128,7 +2130,12 @@ func (h *Handler) siteSessionsGet(w http.ResponseWriter, r *http.Request, p http
 
 	sessions := make([]session.Session, 0, len(trackers))
 	for _, tracker := range trackers {
-		if tracker.GetSessionKind() == types.SSHSessionKind && tracker.GetState() != types.SessionState_SessionStateTerminated {
+		// Only return ssh and k8s session type.
+		if tracker.GetSessionKind() != types.SSHSessionKind && tracker.GetSessionKind() != types.KubernetesSessionKind {
+			continue
+		}
+
+		if tracker.GetState() != types.SessionState_SessionStateTerminated {
```
Comment (xacrimon) [inline lib/web/apiserver.go:2138]: Would it make more sense to assign the kind to a variable above to make this easier to read? That may also make it compact enough to move the state check into this skip conditional.

**Review by xacrimon — APPROVED** [review]

File: lib/session/session.go [hunk 2]
```diff
@@ -74,6 +74,8 @@ func NewID() ID {
 // Session is an interactive collaboration session that represents one
 // or many SSH session started by teleport user
```
Comment (ibeckermayer) [inline lib/session/session.go:75]:
```suggestion
// or many SSH or Kubernetes sessions started by the teleport user
```

**Review by ibeckermayer — APPROVED** [review]

File: lib/session/session.go (hunk 2 above)
Comment (kimlisa) [reply lib/session/session.go]: modified slightly differently since this will probably be used for other types of sessions (maybe)

**Review by zmb3 — APPROVED** [review]
Looks good to me.

**Pull request merged**